src/Client.php

```diff
@@ -83,6 +83,8 @@ class Client
 
     private WriteConcern $writeConcern;
 
+    private bool $autoEncryptionEnabled;
```

Comment:
@alcaeus @GromNaN This new property will not be documentated in Database and Collection options.